identity-iota/src/error.rs

```diff
@@ -51,4 +51,8 @@ pub enum Error {
   CannotRemoveAuthMethod,
   #[error("Cannot Revoke Verification Method")]
   CannotRevokeMethod,
+  #[error("No Client Nodes Provided")]
+  NoClientNodesProvided,
+  #[error("No Explorer for Private Tangles")]
+  NoExplorerForPrivateTangles,
```

Comment:
It's actually possible to deploy an [explorer for private tangles](https://github.com/iotaledger/one-click-tangle/blob/chrysalis/explorer/README.md), so I can add the option to set an explicit explorer url on the `Network`.